src/scripts/main.js

```diff
@@ -1,3 +1,82 @@
 'use strict';
 
-// write code here
+const MAX_COUNT = 10;
+const MIN_COUNT = 2;
+
+const addColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+const addRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+
+function addColumn() {
+  const table = document.querySelector('.field').querySelector('tbody');
+  const rows = table.rows;
+
+  [...rows].forEach((elem) => {
```

Comment:
Here is no need to use spread operator and create a new array, because `forEach` method is just a side-effect method.